src/directives/utils.php

```diff
@@ -1,13 +1,32 @@
 <?php
+/**
+ * Utility functions.
+ *
+ * @package wp-directives
+ */
 
+/** Include WP_Directives_Store class */
 require_once  __DIR__ . '/class-wp-directive-store.php';
 
+/**
+ * Add data to the store.
+ *
+ * @param array $data Data.
+ */
 function wp_store( $data ) {
 	WP_Directive_Store::merge_data( $data );
 }
 
-// TODO: Implement evaluation of complex logical expressions.
-function evaluate( string $path, array $context = array() ) {
+/**
+ * Obtain data from store and context from the provided path.
+ *
+ * TODO: Implement evaluation of complex logical expressions.
```

Comment:
Could use PHPDoc `@todo`

```suggestion
 * @todo Implement evaluation of complex logical expressions.
```